### ISSUE TITLE
Make changes to Spanish and Tagalog language assistance pages in Drupal

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#55707]
## Issue Description

Ana reviewed our current [Spanish language assistance page](https://www.va.gov/asistencia-y-recursos-en-espanol/) for consistency, outdated links, and any errors in ticket #53777. We need to make these changes on the Drupal page, review and publish. 

We need to update the outdated links on the Tagalog page as well.

Word doc of Spanish page with her changes: [https://app.zenhub.com/files/133843125/05983c88-8923-4c98-8493-107ab38a2ee6/download](https://app.zenhub.com/files/133843125/05983c88-8923-4c98-8493-107ab38a2ee6/download)

## Tasks
- [x] Update Spanish page in Drupal (Ana)
- [x] Update links on Tagalog page (Ana)
- [x] Review Spanish page (Megan)
- [x] Review Tagalog page (Megan)
- [x] Publish Spanish page (Megan)
- [x] Publish Tagalog page (Megan)

<details>
<summary>Prepublishing checklist</summary>

### Before you begin
- [ ] Confirm that there are no updates in draft by other content authors that aren't ready to be published. Check the revision log and then confirm with the content author who made the previous edits that the page is ready to publish.

### Accessibility standards

- [ ] If your link or button will take the user off VA.gov, confirm that the text for the link or button contains a purpose and a target. The user should be able to tell from the link or button text what the call to action is (the purpose) and where they will end up if they click on the link (the target). The following link is an example of this: <br>[Read more about making links accessible on AccessibilityOz](https://www.accessibilityoz.com/2014/02/links-and-accessibility/)</br> 
**Note:** When you direct the user to another VA.gov page, the link only needs to contain text indicating purpose (the call to action).
- [ ] Confirm all phone numbers are in aria labels. [Learn how to code numbers with aria labels](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers)
- [ ] Confirm that the hierarchy of section and subsection titles is correct (H1 to H2 to H3—levels shouldn't be skipped). [Learn more about section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- [ ] Replace any sighted language ("view," "see," "look") with a word that's inclusive of our screen reader users ("check," "go to," "review").
- [ ] Avoid using other ability-focused language. ("hear back from")

### Link standards

- [ ] Confirm that links work (aren't broken—no error message) and point to the intended page (you pasted in the correct URL or chose the correct node ID).
- [ ] Confirm all link text is 100 characters or less.
- [ ] Check that you've selected a node ID for all links pointing to VA.gov (internal) pages. Only paste in a URL if the page you're linking to doesn't live in our CMS (isn't one of our modernized pages). [Learn how to add node IDs in Drupal](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/how-to-do-different-tasks/linking-with-node-ids.md) 
- [ ] Check that all anchor/jump links are pointing to a node followed by # and the header identifier (Example: /node/47278/#information-for-survivors). Double check all anchor/jump links post-publish. 

### CMS standards

- [ ] Make sure all apostrophes and quotation marks are curly not straight. (Once this is automatic in Drupal, we can remove this item.)
- [ ] If there are alerts at the top of the Drupal Edit node for broken links, redirected links, or links to unpublished pages, correct the issues. If there's an alert for a broken link but the link is actually working, report the issue to the CMS help desk. [Read more about link alerts in the CMS knowledge base](https://prod.cms.va.gov/help/cms-basics/how-to-fix-broken-links-and-redirects)
- [ ] If publishing a brand-new page, confirm the left nav and url are set up correclty. (If the page is a child page, the section needs to be the same as the parent page in order for the left nav to show.) [Learn how to set up a url and left nav in Drupal](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/how-to-do-different-tasks/adding-left-nav-and-url.md)
- [ ] Check if the page has a new or updated alert. If so, publish the alert separately. 

### Additional checks for translated articles
- [ ] If we don't have updated translated versions, add the alert that links to the corresponding English page and lets users know that the translated pages aren't up to date. 

</details>

## Acceptance Criteria
- [x] Spanish and Tagalog pages on live site are up to date

**Comment by ABeltran77** [comment]
I have some questions regarding continuity and verbiage re: "download" "de/del".

**Comment by ABeltran77** [comment]
Updates have been made to the Spanish page in Drupal. Do we have the link to the English and Tagalog pages?

**Comment by ABeltran77** [comment]
@RLHecht @strelichl Completed the following:

Fixed broken link in Spanish page (link was not in Tagalog page)

These accordions are  not in the Tagalog page, only the Spanish page: 

1. Solicitud e información de cuidado médico del VA
2. Beneficios para Veteranos y miembros del servicio 
-     Información general de beneficios
-     Veteranos con pocos ingresos
-     Veteranos con discapacidades relacionadas al servicio
-     Seguro
-     Preparación y empleo para Veteranos

**Comment by RLHecht** [comment]
@coforma-terry @strelichl, it seems like the work on this has stalled. Can we please prioritize to get this work completed asap? Thanks. FYSA @DanielleThierryUSDSVA

**Comment by ABeltran77** [comment]
@strelichl @coforma-terry The Spanish has been completed. The tagalog pages indicate they cannot be found and the PACT Act link will need to be reviewed to see where it needs to be placed as it does not have a reference section. We can discuss this today in our sync @RLHecht

**Comment by ABeltran77** [comment]
Thank you @RLHecht for your help with the links and for looking into the broken link on the Spanish page.
@RLHecht @strelichl The Tagalog page has been updated but, we are needing a translation for the "Health section" accordion and "More resources" in order to add the PACT Act info. Attached is a word doc for State translators. 
[VA Tagalog wika mapagkukunan at tulong_AB_resources.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/13891570/VA.Tagalog.wika.mapagkukunan.at.tulong_AB_resources.docx)

**Comment by RLHecht** [comment]
Thanks @ABeltran77 A couple of things:

- I was able to fix the broken link on the Spanish page (it wasn't the 10-10ez text that had the broken link but the 10-0137 text linking to an incorrect 10-10ez form). 
- I changed the links you added so that they would be linked by node ID instead of the full url. This is to prevent future broken links if the url ever changes. When linking within drupal we always want to use a node ID.
- I moved several of the links you added under health care to other sections that were more relevant. I also created a few new H2s for the links that didn't fit under specific benefits (accessibility and sexual harassment). I'm waiting for Danielle to review these to see if what I did works or if we should move them elsewhere.
- There is a broken link on both the Spanish and Tagalog page for a community care fact sheet. I asked Danielle whether we should remove it or follow up with a contact there for a new link. I tried searching online for it but wasn't having much luck.
- For the Tagalog page, I just did a quick google translate on the accordion labels and we do already have one that is showing health care so I added the PACT Act link under that. So we don't need to send any new content to State for Tagalog translation.

Once I hear back from @DanielleThierryUSDSVA  on these few things we can pass it to Megan for a QA. Thanks!
